tests/python/PycodestyleBearTest.py

```diff
@@ -17,6 +17,9 @@ def hello():
     print("hello world")
 '''
 
+good_file2 = ('def ' + 'h' * 1000 + '():\n' +
```

Comment:
hmm calling this file "good" is maybe a bit too much... if possible name it something like `file_with_very_long_line` (you get the idea :+1:) :smile: